app/models/article.rb

```diff
@@ -42,17 +42,19 @@ def has_friendly_id_slug?
   ARCHIVAL = "Outdated & ignored in searches."
 
   scope :archived, -> { where.not(archived_at: nil) }
-  scope :current, -> { where(archived_at: nil).order(rotted_at: :desc).order(updated_at: :desc).order(created_at: :desc) }
+  scope :current, -> do
+    where(archived_at: nil)
+      .order(rotted_at: :desc, updated_at: :desc, created_at: :desc)
+  end
   scope :fresh, -> do
-    where("updated_at >= ?", FRESHNESS_LIMIT.ago).
-      where(archived_at: nil).
-      where(rotted_at: nil)
+    where(arel_table[:updated_at].gteq(FRESHNESS_LIMIT.ago))
```

Comment:
Since [arel is a private API](https://twitter.com/tenderlove/status/527604095738060800), I'm not okay with this one. I don't love fragments either, but this is not code I'll have to fix if/when `arel_table` is deprecated or changed.